Samples/Islands/DrawingIsland/DrawingIslandComponents/DrawingIsland.h

```diff
@@ -244,7 +248,8 @@ namespace winrt::DrawingIslandComponents::implementation
         struct 
         {
             winrt::VisualCollection Visuals{ nullptr };
```

Comment:
Since you're adding more members, can you start to add comments about what each of them to? For example, what's the different between Visuals and VisualItems? Do we want to just call "VisualItem" as "Item" to avoid any confusion?

Reply:
OK.